models/articles.models.js

```diff
@@ -1,72 +1,103 @@
-const db = require("../db/connection")
+const db = require("../db/connection");
 
 exports.selectArticleById = (article_id) => {
-    return db.query(`SELECT * FROM articles
-    WHERE article_id = $1`, [article_id])
+  return db
+    .query(
+      `SELECT * FROM articles
+    WHERE article_id = $1`,
+      [article_id]
+    )
     .then(({ rows }) => {
-        if (rows.length === 0) {
-            return Promise.reject({status: 404, msg: 'Article not found'})
-        } else {
-        return rows[0]
-        }
-    })
-}
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Article not found" });
+      } else {
+        return rows[0];
+      }
+    });
+};
+
+exports.selectArticles = (topic) => {
+  const validTopic = ["cats", "mitch"];
```

Comment:
New topics could be added so your code would need to be more dynamic than this. Could you write a model function that checks for the existence of a topic?